modules/mono/editor/bindings_generator.cpp

```diff
@@ -1280,7 +1280,7 @@ Error BindingsGenerator::generate_glue(const String &p_output_dir) {
 
 		List<InternalCall> &custom_icalls = itype.api_type == ClassDB::API_EDITOR ? editor_custom_icalls : core_custom_icalls;
 
-		OS::get_singleton()->print(String("Generating " + itype.name + "...\n").utf8());
+		OS::get_singleton()->print("%s", String("Generating " + itype.name + "...\n").utf8().get_data());
```

Comment:
Why is it necessary to do this? Also, if we are going to do this, wouldn't it be better this way?:

``` C++
OS::get_singleton()->print("Generating %s...\n", itype.name.utf8().get_data());
```

Reply:
Agree with that, thanks.